tests/modules/users/resources/test_modifying_users_info.py

```diff
@@ -417,7 +420,7 @@ def cleanup():
         }
         response = flask_app_client.patch(*args, **kwargs)
         assert response.status_code == 200, response.data
-        response_asset_guid = response.json['profile_fileupload_guid']
+        response_asset_guid = response.json['profile_fileupload']['guid']
```

Comment:
and this is why I'm a big fan or write-it-once-use-it-many-times programming a'la the module utils.
Users doesn't have one and I think this proves it needs one